File: lol/src/main/kotlin/com/your/lol/dto/statistics/ChampionDataDto.java
package com.your.lol.dto.statistics;

import lombok.*;

@Data
@NoArgsConstructor(access = AccessLevel.PRIVATE)
@AllArgsConstructor
public class ChampionDataDto {

    private String championName;
    private double winRate;
    private int matchCount; // 경기 횟수
    private double averageKda;
}

File: lol/src/main/kotlin/com/your/lol/dto/statistics/ChampionDataDto.kt
package com.your.lol.dto.statistics

class ChampionDataDto(
        val championName: String?,
        val winRate: Double,
        val matchCount: Int, // 경기 횟수

        val averageKda: Double) {

    private constructor() : this("", 0.0, 0, 0.0)
}